app/main.py

```diff
@@ -1 +1,48 @@
-#  write your code here
+class Animal:
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True
+                 ) -> None:
```

Comment:
```suggestion
    def __init__(
             self,
                 name: str,
                 appetite: int,
                 is_hungry: bool = True
        ) -> None:
```
re-format it accordingly to the best practices PEP 08